aws/resource_aws_dynamodb_global_table_test.go

```diff
@@ -0,0 +1,259 @@
+	tableName := fmt.Sprintf("tf-acc-test-%s", acctest.RandString(5))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckSesTemplateDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccDynamoDbGlobalTableConfig_basic1(tableName),
+			},
+
+			resource.TestStep{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccCheckAwsDynamoDbGlobalTableDestroy(s *terraform.State) error {
+	conn := testAccProvider.Meta().(*AWSClient).dynamodbconn
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "aws_dynamodb_global_table" {
+			continue
+		}
+
+		input := &dynamodb.DescribeGlobalTableInput{
+			GlobalTableName: aws.String(rs.Primary.ID),
+		}
+
+		_, err := conn.DescribeGlobalTable(input)
+		if err != nil {
+			if isAWSErr(err, "GlobalTableNotFoundException", "") {
+				return nil
+			}
+			return err
+		}
+
+		return fmt.Errorf("Expected DynamoDB Global Table to be destroyed, %s found", rs.Primary.ID)
+	}
+
+	return nil
+}
+
+func testAccCheckAwsDynamoDbGlobalTableExists(name string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		_, ok := s.RootModule().Resources[name]
+		if !ok {
+			return fmt.Errorf("Not found: %s", name)
+		}
+
+		return nil
+	}
+}
+
+func testAccDynamoDbGlobalTableConfig_basic_dynamodb_tables(tableName string) string {
+	return fmt.Sprintf(`
+provider "aws" {
+  alias  = "us-east-1"
+  region = "us-east-1"
+}
+
+provider "aws" {
+  alias  = "us-east-2"
+  region = "us-east-2"
+}
+
+provider "aws" {
+  alias  = "us-west-2"
+  region = "us-west-2"
+}
+
+resource "aws_dynamodb_table" "us-east-1" {
+  provider = "aws.us-east-1"
+
+  hash_key         = "myAttribute"
+  name             = "%s"
+  stream_enabled   = true
+  stream_view_type = "NEW_AND_OLD_IMAGES"
+  read_capacity    = 1
+  write_capacity   = 1
+
+  attribute {
+    name = "myAttribute"
+    type = "S"
+  }
+}
+
+resource "aws_dynamodb_table" "us-east-2" {
+  provider = "aws.us-east-2"
+
+  hash_key         = "myAttribute"
+  name             = "%s"
+  stream_enabled   = true
+  stream_view_type = "NEW_AND_OLD_IMAGES"
+  read_capacity    = 1
+  write_capacity   = 1
+
+  attribute {
+    name = "myAttribute"
+    type = "S"
+  }
+}
+
+resource "aws_dynamodb_table" "us-west-2" {
+  provider = "aws.us-west-2"
+
+  hash_key         = "myAttribute"
+  name             = "%s"
+  stream_enabled   = true
+  stream_view_type = "NEW_AND_OLD_IMAGES"
+  read_capacity    = 1
+  write_capacity   = 1
+
+  attribute {
+    name = "myAttribute"
+    type = "S"
+  }
+}`, tableName, tableName, tableName)
+}
+
+func testAccDynamoDbGlobalTableConfig_basic1(tableName string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "aws_dynamodb_global_table" "test" {
+	depends_on = ["aws_dynamodb_table.us-east-1"]
```

Comment:
Can you fix indentations?

Reply:
Sorry about that! VS Code thinks I want to continue with tabs to match the rest of the file instead of noticing its inside backticks 🙁